.github/workflows/trigger_prerelease.yml

```diff
@@ -0,0 +1,18 @@
+name: Trigger prerelease creation
+
+# This workflow triggers a prerelease creation with changelog and the release notes created by the release toolkit.
+# This workflow should be triggered merely from the default branch.
+# If you wish to be 100% free creating a prerelease, just create it manually.
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 15 * * 2"
+
+jobs:
+  prerelease:
+    uses: newrelic/coreint-automation/.github/workflows/trigger_prerelease.yaml@v1
+    secrets:
+      bot_token: "${{ secrets.COREINT_BOT_TOKEN }}"
```

Comment:
Just adding a comment as a reminder to add secrets :)

Reply:
Thanks for the reminder, already added by David yesterday 👍